includes/ets_user_qa_function.php

```diff
@@ -179,7 +180,7 @@ public function ets_ask_qustion_tab() {
 				<div class="ets-dis-message-error"><p></p></div>
 				<button id="ets-submit" type="submit" name="submit" class="btn btn-info" ><?php echo __('Submit','product-questions-answers-for-woocommerce'); ?></button> 
 			</form>
-			<div id="ets_product_qa_length"><p></p></div>   
+		  
```

Comment:
What was the use of statement `<div id="ets_product_qa_length"><p></p></div>` removed from this line.

Reply:
It is used when we put the shortcode and default functionality together when we don't use this it dismisses the shortcode load more.